crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -40,8 +40,24 @@ def role_constraints
   # Unfortunatelly we need to explicitely look at crowbar-status of the proposal
   # because apply_role from this model ignores errors from superclass's apply_role.
   def commit_and_check_proposal
-    proposal_commit("default", false, false)
+    answer = proposal_commit("default", false, false)
+    found_errors        = []
```

Comment:
Style/ExtraSpacing: Unnecessary spacing detected.<br>Style/SpaceAroundOperators: Operator `=` should be surrounded by a single space. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)